src/wp-includes/block-editor.php

```diff
@@ -464,3 +464,47 @@ function block_editor_rest_api_preload( array $preload_paths, $block_editor_cont
 		'after'
 	);
 }
+
+/**
+ * Creates an array of theme styles to load into the block editor.
+ *
+ * @since 5.8.0
+ *
+ * @global array $editor_styles
+ *
+ * @return array
```

Comment:
We'll just want to make sure a `@return` description is added here when this gets committed so that IDEs have something to show for the description.